pages/submissions/[id]/steps/[stepId].tsx

```diff
@@ -50,17 +47,6 @@ const StepPage = ({
     }
   };
 
-  const handleFinish = async (
-    setStatus: (message: string) => void
-  ): Promise<void> => {
-    try {
-      await axios.post(`/api/submissions/${params.id}`);
-      router.push(`/people/${residents[0].id}/submissions/${params.id}`);
-    } catch (e) {
-      setStatus(e.toString());
-    }
-  };
-
```

Comment:
unused function

Reply:
this is from when we were thinkng about doing single-step forms on the same file. now no longer needed!